lib/src/10_auth/auth_screen.dart

```diff
@@ -128,10 +146,45 @@ class _FauiAuthScreenState extends State<FauiAuthScreen> {
     );
   }
 
-  Widget _buildCreateAccountScreen(BuildContext context) {
-    final TextEditingController emailController =
-        new TextEditingController(text: this._email);
+  Widget _buildTextBox(
+      TextEditingController controllerName,
+      FocusNode currentNode,
+      FocusNode nextNode,
+      String fieldName,
+      bool autoFocus,
```

Comment:
I suggest to remove this parameter and have autoFocus always true, for simplicity.

Reply:
done